records_mover/records/__init__.py

```diff
@@ -9,8 +9,11 @@
     'ParquetRecordsFormat',
     'ProcessingInstructions',
     'ExistingTableHandling',
+    'RecordsFolderNonEmptyException',
+    'RecordsException',
     'Records',
-    'move'
+    'move',
+    'MoveResult'
```

Comment:
Let's bring more things so they can be pulled in with a simple import from `records_mover.records`.